core/labeled-replies-3.3.md

```diff
@@ -0,0 +1,36 @@
+---
+title: IRCv3.3 Labeled replies
+layout: spec
+copyrights:
+  -
+    name: "Alexey Sokolov"
+    period: "2015"
+    email: "alexey-irc@asokolov.org"
+---
+Name of this capability is `labeled-replies`.
+If client requests `labeled-replies`, it MUST also request `batch` capability.
+
+If the capability is enabled, client MAY tag its messages with `label` tag.
+
+For messages from client which are tagged with `label`, server MUST send exactly
+one message tagged with the same `label` value in response to such message.
+If response contains more than one message, `batch` can be used to group them.
+In that case start of the batch MUST be tagged with the label, and the batch type MUST be `labeled-response`.
+
+If the message doesn't require any response, an empty batch MUST be sent.
```

Comment:
What does this mean? Is it something like, if `echo-message` isn't enabled, labeling privmsgs would result in something like this?

```
Client: @label=somelabel PRIVMSG #channel :Hello!
Server: @label=somelabel :irc.example.com BATCH +somebatch
Server: :irc.example.com BATCH -somebatch
```